AzFilesHybrid/AzFilesHybrid.psm1

```diff
@@ -2144,8 +2142,8 @@ function Ensure-KerbKeyExists {
                 Write-Error -Message "Caught exception: $_" -ErrorAction Stop
             }
 
-            $kerb1Key = Get-AzStorageAccountKey -ResourceGroupName $ResourceGroupName -Name $StorageAccountName `
-                 -ListKerbKey | Where-Object { $_.KeyName -eq "kerb1" }
+            $kerb1Key = Get-AzStorageAccountKerbKeys -ResourceGroupName $ResourceGroupName -StorageAccountName $StorageAccountName `

```

Comment:
Will this log to error log if kerb keys do not exist for the account yet? Join-SA ->Ensure Kerb Keys -> Get-AzStorageAccountKerbKeys -> write-error -Error action stop

Not sure if we should log an error in "sunny' case.

Reply:
Get-AzStorageAccountKerbKerys is protected by a try-catch before New-AzStorageAccountKey is called (at line 2125). After New-AzStorageAccountKey is called, Get-AzStorageAccountKerbKerys should fail if no kerb keys are found.